roles/nova_compute_drain/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+
+nova_compute_drain_venv: "{{ virtualenv_path }}/openstack"
```

Comment:
virtualenv_path is defined by kayobe. Should I add some default here? Something in the home directory maybe?